src/features/currentTodo.ts

```diff
@@ -28,7 +28,11 @@ const currentTodoReducer = (
   action: Action,
 ): State => {
   switch (action.type) {
-    // Implement all actions here
+    case 'currentTodo/SET':
+      return action.payload;
+
+    case 'currentTodo/REMOVE':
```

Comment:
Create an enum for action types. It is not very good to use just strings, as you can make a mistake in one letter and the code will break.